src/bundle/Resources/views/Limitation/udw_limitation_value.html.twig

```diff
@@ -1,27 +1,42 @@
-<div>
+<div class="ez-limitation-picker">
     {{ form_label(form.limitationValues) }}
     {{ form_errors(form.limitationValues) }}
     {{ form_widget(form.limitationValues) }}
 
-    <div>
-        <button
-            data-universaldiscovery-title="{{ "role.policy.limitation.location.udw_title"
-            |trans({}, "ezrepoforms_role")
-            |desc("Select locations to use as policy limitation") }}"
-            class="ez-button-tree pure-button ez-font-icon ez-button btn btn-secondary ez-pick-location-limitation-button"
-            data-location-input-selector="#{{ form.limitationValues.vars.id }}"
-            data-selected-location-list-selector="#{{ form.limitationValues.vars.id }}-selected-location"
-            data-udw-config="{{ ez_udw_config('multiple', {}) }}"
-        >{{ "role.policy.limitation.location.udw_button"|trans({}, "ezrepoforms_role")|desc("Select locations") }}</button>
-    </div>
-</div>
+    <button type="button" data-universaldiscovery-title="{{ "role.policy.limitation.location.udw_title"
+        |trans({}, "ezrepoforms_role")
+        |desc("Select locations to use as policy limitation") }}"
+        class="ez-button-tree pure-button ez-font-icon ez-button btn btn-secondary ez-pick-location-limitation-button d-block"
```

Comment:
Please, remove the `pure-button` and other unused classes from here and other places.